docs/getting_started/creating_a_flyte_project.md

```diff
@@ -0,0 +1,79 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+# Creating a Flyte project
```

Comment:
Currently, "Flyte project" refers to two completely different things, which is very confusing. Ideally, we would rename one, but I'm not sure how feasible that is.